postreise/plot/plot_energy_carbon_stack.py

```diff
@@ -31,7 +31,7 @@ def plot_n_scenarios(*args):
     plant = {k: v.plant for k, v in grid.items()}
     # First scenario is chosen to set fuel colors
     type2color = ModelImmutables(args[0].info["grid_model"]).plants["type2color"]
-    carbon_by_type, energy_by_type, type2color = {}, {}
+    carbon_by_type, energy_by_type = {}, {}
```

Comment:
Nice

Reply:
That is a bug I introduced when I refactored the modules in PostREISE to use the grid model information.